test/modules/environment.js

```diff
@@ -49,10 +49,17 @@ module.exports = {
   },
 
   getSpectronApp() {
-    return new Application({
+    const options = {
       path: electronBinaryPath,
       args: [`${path.join(sourceRootDir, 'src')}`, `--data-dir=${userDataDir}`, '--disable-dev-mode'],
-    });
+
+      // enable this if chromedriver hangs to see logs
+      // chromeDriverLogPath: '../chromedriverlog.txt',
+    };
+    if (process.platform === 'darwin') {
```

Comment:
Can you describe what this part of the change enables?

Reply:
I added a comment :) thanks for pointing out it wasn't clear